src/sidebar/test/postmessage-json-rpc-test.js

```diff
@@ -10,7 +10,7 @@ class FakeWindow {
   }
 }
 
-describe('sidebar.util.postmessage-json-rpc', () => {
+describe('sidebar/postmessage-json-rpc', () => {
```

Comment:
`postmessage-json-rpc` is a good candidate for a generic util as well IMO, since it has no dependencies and can work with any service that implements the JSON-RPC protocol over `postMessage`.